pygame_life.py

```diff
@@ -15,22 +106,20 @@
 
 
 def get_neighbours(grid: Grid, x: int, y: int) -> Neighbours:
-    """
-        Gets the neighbour states for a particular cell in
-        (x, y) on the grid.
-    """
     offsets = [(-1, -1), (0, -1), (1, -1), (-1, 0), (1, 0), (-1, 1), (0, 1), (1, 1)]
     possible_neighbours = {(x + x_add, y + y_add) for x_add, y_add in offsets}
     alive = {(pos[0], pos[1]) for pos in possible_neighbours if pos in grid.cells}
     return Neighbours(alive, possible_neighbours - alive)
+    
 
 
 def update_grid(grid: Grid) -> Grid:
+    new_cells = deepcopy(grid.cells)
     """
         Given a grid, this function returns the next iteration
-        of the game of life.
+        of the game of life.    

```

Comment:
Remove the extra spaces after the last char

Reply:
Nice, but **unnoticeable**.